README.md

```diff
@@ -86,6 +86,66 @@ To enable verbose debug mode use:
 ./build/src/edgesec -c ./build/dev-config.ini -ddddd
 ```
 
+## Docker for cross compile
+
+The following commands also work using the `podman` container runtime (recommended on Linux):
+
+```console
+alias docker=podman
+```
```

Comment:
`console` should only be used if you have a syntax like the following (e.g. with PS1, command, and stdout/stderr):

```console
user@server:/folder-path/to/dir$ echo "hello"
hello
```

If it's just the command, you normally want `bash` or `sh`

`````suggestion
```bash
alias docker=podman
```
`````